jenkins/scripts/integration_test.sh

```diff
@@ -44,7 +44,7 @@ if [[ "${TESTS_FOR}" == "feature_tests"* || "${TESTS_FOR}" == "e2e_tests"* ]]
 then
   RESIZED_VM_SIZE="${RESIZED_VM_SIZE:-300}"
 else
-  RESIZED_VM_SIZE="${RESIZED_VM_SIZE:-100}"
+  RESIZED_VM_SIZE="${RESIZED_VM_SIZE:-200}"
```

Comment:
I don't think we use RESIZED_VM_SIZE anywhere anymore, since we don't boot from volume. We can remove this block of code entirely.

Reply:
Thanks, I have removed it.